src/main/java/org/folio/service/BatchVoucherExportsService.java

```diff
@@ -55,12 +55,14 @@ public void deleteBatchVoucherExportsById(String id, Context vertxContext, Handl
 
   public Future<Tx<Map<String, String>>> deleteBatchVoucherExportById(Tx<Map<String, String>> tx) {
     Promise<Tx<Map<String, String>>> promise = Promise.promise();
-
```

Comment:
I suggest to move tx.getEntity().get(BATCH_VOUCHER_EXPORT_ID) to separate variable

Reply:
Ok, I will fix